io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,33 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+        minimal:
+            threads: 5
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+        minimal:
+            numruns: 1
+  
+    iteration: !mux
```

Comment:
PS: In the future I can imagine a new tag to produce those automatically, eg:

```yaml
iterations:
    quick:
        !range : 2
```

(that would be actually very simple change in avocado.core.tree, like 3-4 lines...)